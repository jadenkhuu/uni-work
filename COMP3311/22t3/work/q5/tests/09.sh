python3 ./q5 "Moonee Valley" 2019-10-25
