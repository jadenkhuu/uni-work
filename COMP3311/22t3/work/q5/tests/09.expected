Race meeting at Moonee Valley on 2019-10-25

Race 1, prize pool $5000, run over 1700m
Richard Friction ridden by Martin Garcia wins $3500
St. Frusquin ridden by Eddie Castro wins $1000
Russel ridden by Yoshitomi Shibata wins $500

Race 2, prize pool $10000, run over 2100m
Donovan ridden by Kirsty Milczarek wins $7000
Firdaussi ridden by Pesi Shroff wins $2000
Hoof Hearted ridden by Tommy Weston wins $1000

Race 3, prize pool $7000, run over 1400m
Fairway ridden by Martin Molony wins $4900
Felstead ridden by Kieren Fallon wins $1400
Mane Event ridden by Nina Carberry wins $700

Manikato Stakes, prize pool $1000000, run over 1200m
Ladas ridden by Alicia Thornton wins $700000
Shotover ridden by Manuel Ycaza wins $200000
Talk Derby to Me ridden by Sandy Hawley wins $100000
